MDM.py

```diff
@@ -72,134 +77,139 @@ def mean_riemann(covmats, tol=10e-9, maxiter=50, init=None,
-    return np.sqrt((np.log(eigvalsh(A, B))**2).sum())
-def dist_riem(A,B):
-    return np.linalg.norm(scipy.linalg.logm(np.dot(np.linalg.inv(A) , B)))
+    return np.sqrt((np.log(eigvalsh(A, B)) ** 2).sum())
+
+
+def dist_riem(A, B):
+    return np.linalg.norm(scipy.linalg.logm(np.dot(np.linalg.inv(A), B)))
+
+
+# Proper code
+
 
-"""Tutaj som właściwe perełki"""
 class Base(object):
-#   Ten Pan potrzebuje dostać 4-ro wymiarowego arraya z taką mapką indeksów:
-#   (x,y,z,s) - to niech bedzie rozmiar
-#       x - to będzie mówic ile mamy gestów
-#       y - to mówi ile powtórzeń w każdym geście(musi ich być tyle samo dla każdego gestu)
-#       z - tutaj mamy liczbe kanałów
-#       s - liczba próbek sygnału jakie dostajemy (gługość sygnału wyrażona w próbkach)
-#   Jeśli Pan dostanie wszystko co mu do szczęścia potrzebne to zwróci trójwymiarową macierz z średnimi odległościami
-#   do której będziemy się odnosić przy porównywaniu sygnału/gestu live
-    def __init__(self,arr):
-        self.__gesty = arr
-        self.__ilegestow = arr.shape[0]
-        self.__ilepow = arr.shape[1]
+    #   Accepts 4 dimensional array:
+    #   (x,y,z,s) - let it be the dimension
+    #       x - how many gestures are to be classified
+    #       y - number of repetitions for each gesture (equal for every gesture)
+    #       z - number of channels
+    #       s - number of samples in each gesture
+    #   Returns 3 dimensional array with gestures classified based on riemann geometry
+    #   which is necessary for classifying gestures with online signal
+    def __init__(self, arr):
+        self.__gestures = arr
+        self.__number_of_gestures = arr.shape[0]
+        self.__repetitions = arr.shape[1]
         self.__ch = arr.shape[2]
-        self.__SPDmatrices = np.zeros((self.__ilegestow,self.__ch,self.__ch))
+        self.__SPDmatrices = np.zeros((self.__number_of_gestures, self.__ch, self.__ch))
+
+    #   matrix with matrices classified (one gesture - one 2 dimensional matrix)
     def Make_SPDBase(self):
-        CovPackage = np.zeros((self.__ilegestow , self.__ilepow , self.__ch , self.__ch))
-#       macierz do której będzieny wsadzać kowariancje pojedynczych gestów        
-#        SPDmatrices = np.zeros((self.__ilegestow,self.__ch,self.__ch))
-#       macierz 3d do której będziemy wkładać macierze 2d opisujące średnią odległość między macierzami kowariancji dla powtórzeń danego gestu
-        for index in np.ndindex(self.__ilegestow,self.__ilepow):
-            #w tej pętli liczmy macierz kowariancji wszystkich powtórzeń po kolei
-            CovPackage[index[0],index[1],:,:] = np.corrcoef(self.__gesty[index[0],index[1],:,:])
-        for gest in range(self.__ilegestow):
-            # w tej pętli liczymy średnie odległości
-            mean = mean_riemann(CovPackage[gest,:,:,:], tol=1e-08, maxiter=50, init=None, sample_weight=None)
-            self.__SPDmatrices[gest,:,:] = mean[:,:]
+        CorrPackage = np.zeros(
```

Comment:
If possible stick with one convention in variables naming.

    Camel Case: Second and subsequent words are capitalized, to make word boundaries easier to see. 
        Example: corrPackage
    Pascal Case: Identical to Camel Case, except the first word is also capitalized.
        Example: CorrPackage
    Snake Case: Words are separated by underscores.
        Example: corr_package